float32 period_secs
float32 stop_time
---
bool success
